app/sessionStore.js

```diff
@@ -254,12 +256,7 @@ module.exports.cleanAppData = (data, isShutdown) => {
   if (data.sites) {
     const clearHistory = isShutdown && getSetting(settings.SHUTDOWN_CLEAR_HISTORY) === true
     if (clearHistory) {
-      // TODO - this should the history methods from siteUtils
-      data.sites = data.sites.filter((site) => site && site.tags && site.tags.length)
-      data.sites = data.sites.map((site) => {
-        delete site.lastAccessedTime
-        return site
-      })
+      data.sites = siteUtil.clearHistory(Immutable.fromJS(data.sites)).toJS()
```

Comment:
I don't think it's a big deal here, it's just a single call in an infrequently called method